src/GafferSceneModule/FilterBinding.cpp

```diff
@@ -63,6 +63,12 @@ ScenePlugPtr getInputScene( const Gaffer::Context *context )
 	return const_cast<ScenePlug *>( Filter::getInputScene( context ) );
 }
 
+int match( const FilterPlug &plug, const ScenePlug &scene )
+{
+	return plug.match( &scene );
```

Comment:
This triggers computes, and therefore needs to be inside a `ScopedGILRelease`.

Reply:
Good spot thanks, added [here](https://github.com/GafferHQ/gaffer/pull/4057/commits/ecfd13b73c42d945ec339d5f4b7c835934fcec2b).